sources/packages/backend/apps/db-migrations/src/sql/SFASRestrictions/Add-col-is-included.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+  sims.sfas_restrictions
+ADD
+  COLUMN processed BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
processed means processed in SIMS right?

Reply:
Yes, `processed` means inserted in sims `student_restrictions` table.

Reply:
Not really, `processed` means that it was checked. There is a possibility to a restrictions be created in the student account but not every single SFAS record, even belonging to a SIMS student, will result in a new restriction inserted.